composer.json

```diff
@@ -5,18 +5,21 @@
     "keywords": ["rabbitmq", "queue"],
     "license": "GPL-3.0",
     "require": {
-        "php": "^7.1",
-        "enqueue/amqp-bunny": "^0.8.40",
-        "guzzlehttp/guzzle": "^6.3",
-        "pnz/json-exception": "^0.1.0",
-        "symfony/console": "^4.1"
+        "php": "^7.4 || ^8.0",
+        "enqueue/amqp-bunny": "^0.10.9",
+        "guzzlehttp/guzzle": "^6.5.8",
+        "pnz/json-exception": "^1.0.0",
+        "symfony/console": "^5.4"
```

Comment:
Would it be possible to allow version 6 here as well, e.g. `^5.4 || ^6.0`?

Reply:
Same for Guzzle , I guess.

Reply:
my thinking was that we build and deploy for php 7.4 and symfony 6 does not support php 7 anymore. 

but we could still allow it for people checking out the code and building the phar themselves on php 8.

Reply:
guzzle would probably work, but i'd rather keep the change to a separate pull request to not change too much at the same time.